### PR TITLE
Add PHPoole.

### PR DESCRIPTION
**Pull request by Narno**

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -6352,6 +6352,17 @@
...
 			"website": "https://phenomic.io/"
 		},
+    "PHPoole": {
+			"cats": [
+				"1",
+				"11"
+			],
+			"icon": "PHPoole.png",
+			"meta": {
+				"generator": "PHPoole"
```
Comment (jvoisin) [inline src/apps.json]: Please use "generator": "^PHPoole$"` instead to speed up a bit the matching.

Comment (Narno) [reply src/apps.json]: Done! :-)

**Comment by Narno** [comment]
Hello @jvoisin, thanks for your feedback: PHPoole's icon is now 64x64! :-)

**Comment by AliasIO** [comment]
Thanks. Do note that the PNG will be invisible for most people as it's white on white.

**Comment by Narno** [comment]
Hello @AliasIO, maybe I should switch to the black version in the future.
Thanks to you and to @jvoisin! :-)

**Pull request merged**